chucks-pick3-web/angular-app/src/app/app-routing.module.ts

```diff
@@ -41,5 +44,10 @@ export const routingComponents = [
   DemoComponent,
   SignUpComponent,
   LogInComponent,
-  SignUpConfirmationComponent
+  SignUpConfirmationComponent,
+  CustomerSummaryComponent,
+  InfoGenerateComponent,
```

Comment:
Interesting that you are just now adding the InfoGenerateComponent.  How was this able to work before?

Reply:
Actually I'm not sure how this wasn't there before, Everytime I would test it the components were there. I think this might have happened after I completed the merge conflicts on it?